src/test/java/SergioR/SeleniumTest1.java

```diff
@@ -0,0 +1,83 @@
+package SergioR;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class SeleniumTest1 {
+
+	public static void main(String[] args) {
+		System.setProperty("webdriver.chrome.driver", "src/test/resources/Drivers/chromedriver");
+		WebDriver driver = new ChromeDriver();
+
+		driver.manage().timeouts().implicitlyWait(5, TimeUnit.SECONDS);
+
+		driver.navigate().to(
+				"file:///Users/pk/eclipse-workspace/Homework/src/main/resources/MuradilE/Bora%20UI%20Test/landing.html");
+
+		WebElement element = driver.findElement(By.xpath("/html/body/nav/div/a"));
+
+		if (element.isDisplayed()) {
+			System.out.println("found: " + element.getText());
+		} else {
+			System.out.println("not found");
+
+		}
+
+		driver.findElement(By.xpath("//*[@id=\"mobile-nav\"]/ul[1]/li/a")).click();
+
+		WebElement stuinfotext = driver.findElement(By.xpath("/html/body/div/div/div/div/h1"));
+
+		if (stuinfotext.isDisplayed()) {
+			System.out.println("found: " + stuinfotext.getText());
+		} else {
+			System.out.println("not found");
+		}
+
+		WebElement table = driver.findElement(By.id("studentInfo"));
+
+		List<WebElement> rows = table.findElements(By.xpath("//*[@id=\"studentInfo\"]/tbody/tr"));
+
+		if ((rows.size() - 1) == 10) {
+			System.out.println("there are 10 students");
+		} else {
+			System.out.println("there are not 10 students");
+		}
+
+		for (int i = 1; i < rows.size(); i++) {
+
+			List<WebElement> student = rows.get(i).findElements(By.tagName("td"));
+
+			System.out.println(student.get(0).getText());
+
+		}
+
```

Comment:
Sergio, from Line 59 to 77, you're kinda assuming that you know exactly where you name is at in the table, and you're explicitly wrote an CSS selector to point directly to email cell and phone number cell. That's not what I'm looking for right here. 

Notice that in line 51-57 you're already looking through the name cells, from which you can get a list of names and find the location of your name, and then with that, you can then locate the other two cells and complete your validation.

Something like: 
```
boolean found = false;
for (int i = 1; i < rows.size(); i++) {
	List<WebElement> student = rows.get(i).findElements(By.tagName("td"));
	// by accessing the text of the first cell of each table row, we get the names
	if (student.get(0).getText().equals("Sergio")) {
		// now it means you found your name in the list, programmatically, not by
		// eyeballing it
		found = true;
		System.out.println("Name - Sergio - is found in the list at position: (" + i + ",1)");
		// so now, since we found the name, we will then continue to validate the email
		// and phone number
		// they are the second and third td element, in this very row we're in, which is
		// i
		if (student.get(1).getText().contains("@bora-cs.com")) {
			// we're only validation if the email text contains @bora-cs.com
			System.out.println("The email is in the xxxxx@bora-cs.com format");
		} else {
			System.out.println("The email format is wrong:");
			System.out.println("Actual email found: " + student.get(1).getText());
		}
		if (student.get(2).getText().contains("202-202-20")) {
			// we're only validation if the phone number contains the first 8 digits
			// 202-202-20
			System.out.println("The phone number is in the 202-202-20** format");
		} else {
			System.out.println("The phone number format is wrong:");
			System.out.println("Actual phone number found: " + student.get(1).getText());
		}
		// and that's it, if the code gets this far, it means the name is found, email
		// is validated, and phone number is validated as well
	}
}
if (!found)
	System.out.println("Name - Sergio - is not found");
```

This above code should replace everything from line 51-77 if it makes sense to you.